digitalidentity/requests/signed_message.go

```diff
@@ -0,0 +1,222 @@
+package requests
```

Comment:
what was the reason for copying this entire package? it's better to have 1 copy of common code so we can maintain it easier and it won't bite us in the ass if we make a fix to one but forget the copy. 

if the whole reason was because we need it to use the new `yotierror` response to error parsing, we can just version the `Execute` function like `Execute2` or something (or if the name is important like it will be used directly by integrators then we could just have this function by itself inside the `digitalidentity` package instead) and have that do the new error handling

Reply:
Good points, but, as we plan to retire V1 I thought this version is preferable. That's the same reason the choice was taken to duplicate the Policy code. See the notes here: https://lampkicking.atlassian.net/browse/SDK-2230.